### PR TITLE
[42] Add automated tests for OpenAPI content types

### LINKED ISSUE #42: Add automated tests for OpenAPI content types
Add automated tests for OpenAPI content types

### PR DESCRIPTION
**Pull request by keithchong**
Add automated tests for OpenAPI content types

Fixes #42 

Signed-off-by: Keith Chong <kchong@ca.ibm.com>

### CODE REVIEW

**Comment by keithchong** [comment]
@sghung , I made two non-test related changes.  I added two string constants to the core plugin.

File: dev/org.eclipse.codewind.openapi.core/src/org/eclipse/codewind/openapi/core/IOpenApiConstants.java [hunk 1]
```diff
@@ -23,4 +23,7 @@
 	public static final String CODEWIND_PROJECT_LANGUAGE = "projectLanguage"; //$NON-NLS-1$
 	public static final String CODEWIND_GET_ID_METHOD = "getId"; //$NON-NLS-1$
 	public static final String CODEWIND_CW_SETTINGS = ".cw-settings"; //$NON-NLS-1$
+	
+	public static String CONTENTTYPE_YAML = "org.eclipse.codewind.openapi.core.openApiDocumentYaml";
```
Comment (sghung) [inline dev/org.eclipse.codewind.openapi.core/src/org/eclipse/codewind/openapi/core/IOpenApiConstants.java]: Do we need to add the `//$NON-NLS-1$` anymore? Or are we stopping that?

Comment (keithchong) [reply dev/org.eclipse.codewind.openapi.core/src/org/eclipse/codewind/openapi/core/IOpenApiConstants.java]: @sghung, added.

**Comment by sghung** [comment]
Looks good to me. This does not work well with WDT though, but WDT support can be added later (changes have to be made on the WDT side)

**Pull request merged**